sample-customisation/src/main/java/net/helpscout/sample/beacon/CustomisationActivity.java

```diff
@@ -1,155 +1,151 @@
+     * more info https://developer.helpscout.com/beacon-2/android/#session-attributes
      */
-    private void addUserAttributes() {
-        Beacon.addAttributeWithKey("App version", getAppVersion());
-        Beacon.addAttributeWithKey("OS version", Build.VERSION.RELEASE);
-        Beacon.addAttributeWithKey("Device", Build.MANUFACTURER + " " + Build.MODEL);
+    private void addSessionAttributes(boolean enabled) {
+        Map attributes = new HashMap<String, String>();
+        if (enabled) {
+            attributes.put("App version", Utils.getAppVersion(this));
+            attributes.put("OS version", Build.VERSION.RELEASE);
+            attributes.put("Device", Build.MANUFACTURER + " " + Build.MODEL);
+        }
+        Beacon.setSessionAttributes(attributes);
     }
 
     /**
      * Pre-fill the Contact us form
+     * More info https://developer.helpscout.com/beacon-2/android/#prefilling-the-contact-form
      */
-    private void addPreFilledData() {
-        Map<Integer, String> prePopulatedCustomFields = new HashMap<>();
-        prePopulatedCustomFields.put(123, "TEST");
-
-        Beacon.addPreFilledForm(new PreFilledForm(
-                "My Secure user Scott",
-                "Bug report for app",
-                "Please include steps to reproduce the issue",
-                prePopulatedCustomFields,
-                generateLogFileUri(),
-                "my@email.com"
-        ));
+    private void addPreFilledData(boolean enabled) {
+        Beacon.contactFormReset();
+        if (enabled) {
+            Map<Integer, String> prePopulatedCustomFields = new HashMap<>();
+            prePopulatedCustomFields.put(123, "TEST");
+
+            ArrayList attachments = new ArrayList(Utils.generateSampleLogFileUri(this));
+
+            Beacon.addPreFilledForm(new PreFilledForm(
+                    "Prefill Name",
+                    "Prefill Subject",
+                    "Please include steps to reproduce the issue",
+                    prePopulatedCustomFields,
+                    attachments,
+                    "prefill@email.com" // email (this can be edited by the user)
+            ));
+        }
     }
 
     /**
-     * Add suggested articles
+     * More info on settings overrides https://developer.helpscout.com/beacon-2/android/#settings-customization
      */
-    private void addArticlesSuggestionOverride() {
-        Beacon.setOverrideSuggestedArticlesOrLinks(articleSuggestionsOverride);
+    private void overrideBeaconConfig(boolean shouldDisableNameSubject, boolean colorOverrideEnabled) {
+
+        //Demonstrates how to use Android Color Resource Hex to override the Beacon color
+        String colorOverride = colorOverrideEnabled ? getResources().getString(R.color.custom_beacon_color) : null;
+
+        Beacon.setConfigOverrides(new BeaconConfigOverrides(
+                true, // docsEnabled
+                true, // messagingEnabled
+                true, // chatEnabled
```

Comment:
🔍  Would it make sense to explain in here that what `[docs|messaging|chat]Enabled` basically means when set to `true` is that the remote config will be used, not that they'll be "enabled"?

Reply:
Good point, I think that extra contect will help.

Reply:
```
// [docs|messaging|chat]Enabled - is primarily designed for runtime disabling of features
// When set to false this will locally override and disable a feature
// When set to true the remote config will be used and does not enable a feature if
// it isn't enabled on the Beacon on Help Scout
```

Reply:
Great suggestion! Do the modified docs reflect this as well? I need to check those next!

Reply:
Test

Reply:
Test